tests/nlu/selectors/test_selectors.py

```diff
@@ -274,6 +275,7 @@ async def _train_persist_load_with_different_settings(
 
 
 @pytest.mark.skip_on_windows
+@pytest.mark.trains_model
```

Comment:
what about `_train_persist_load_with_different_settings`?

Reply:
it's not a test, but I've marked the tests that call it.